doozer/doozerlib/cli/scan_osh.py

```diff
@@ -158,6 +433,16 @@ async def run(self):
 
         self.trigger_scans(nvrs_for_scans)
 
+        if not self.runtime.group_config.scanning.jira_integration.enabled:
```

Comment:
Maybe an explicit check is better here `if enabled in [True, "true", "yes"]`

Reply:
fixed in https://github.com/openshift-eng/art-tools/pull/141/commits/e4031289cbb0f229b3524b08cc04909ef204bc17

Reply:
Don't see it..

Reply:
Fixed and pushed https://github.com/openshift-eng/art-tools/pull/141/files#diff-990549c3908d335c0917a3ed7d6fda9b6069fd0e0551ede2e5838cd87dc6b92cR455